crates/storage/src/tables/slot_index.rs

```diff
@@ -0,0 +1,42 @@
+use std::sync::Arc;
+
+use alloy_primitives::B256;
+use redb::{Database, Durability, TableDefinition};
+
+use super::{SSZEncoding, Table};
+use crate::errors::StoreError;
+
+/// Table definition for the Slot Index table
+///
+/// Key: slot number
+/// Value: block_root
+pub const SLOT_INDEX_TABLE: TableDefinition<SSZEncoding<u64>, SSZEncoding<B256>> =
```

Comment:
```suggestion
pub const SLOT_INDEX_TABLE: TableDefinition<u64, SSZEncoding<B256>> =
```
The `redb` `Key/Value` traits are already implemented for `u64`, so we don't need to wrap this in `SSZEncoding`